x-pack/legacy/plugins/maps/server/routes.js

```diff
@@ -363,7 +363,7 @@ export function initRoutes(server, licenseUid) {
 
   server.route({
     method: 'GET',
-    path: `${ROOT}/${EMS_SPRITES_PATH}/{id}/sprite{scaling}.{extension}`,
+    path: `${ROOT}/${EMS_TILES_API_PATH}/${EMS_SPRITES_PATH}/{id}/sprite{scaling}.{extension}`,
```

Comment:
the `proxyElasticMapsServiceInMaps` setting doesn't seem to work for ems basemaps (?)

![image](https://user-images.githubusercontent.com/1833023/72310138-fdb3f600-364e-11ea-8ff1-6789758a08f0.png)


It looks like the sprite url isn't being supplied correctly. 

e.g.

`http://localhost:5601/fye/api/maps/ems/tiles/sprites/dark_map/sprite.json?elastic_tile_service_tos=agree&my_app_name=kibana&my_app_version=8.0.0&license=...` would fail

It seems the scaling param is missing

something like ``http://localhost:5601/fye/api/maps/ems/tiles/sprites/dark_map/sprite2@.json?elastic_tile_service_tos=agree&my_app_name=kibana&my_app_version=8.0.0&license=...` would work.